packages/rocketchat-i18n/i18n/de.i18n.json

```diff
@@ -1267,12 +1298,19 @@
   "Favorites": "Favoriten",
   "Feature_Depends_on_Livechat_Visitor_navigation_as_a_message_to_be_enabled": "Diese Funktion hängt davon ab, ob \"Besucher-Navigationsprotokoll als Nachricht senden\" aktiviert sein soll.",
   "Features_Enabled": "Aktivierte Funktionen",
+  "FEDERATION_Enabled": "Versuchen Sie, die Verbundunterstützung zu integrieren. Um diesen Wert zu ändern, muss Rocket.Chat neu gestartet werden. ",
+  "FEDERATION_Enabled_Alert": "Federation-Support ist eine Arbeit in Arbeit. Die Verwendung auf einem Produktionssystem wird derzeit nicht empfohlen.",
```

Comment:
"Federation-Support ist eine Arbeit in Arbeit."

Bitte anpassen. Ergibt so keinen Sinn